### DISCUSSION TITLE
Converting my RASCSI to a "backpack style" unit

### DISCUSSION THREAD

**Discussion by dburr** [akuker/RASCSI#752] [Q&A]
I'd like to know if it is possible to convert my RASCSI to a "backpack style" unit -- one where it attaches directly to the SCSI connector on the Mac, without the need for a cable. Is it just as simple as desoldering the female DB25 that's on there now and putting in a male DB25? Can anyone point me to a male DB25 that would fit the existing PCB footprint?

Also, is there any source of power on the Mac itself that would provide enough current to run the RASCSI, obviating the need for a separate USB wall wart to power the thing? (ADB port? floppy port?)

**Comment by jcres69** [comment]
Not sure if this is what you are looking for, but this is what I used to connect directly to the back of my mac plus:

https://www.mcmaster.com/7171K31/
